ecs-agent/tmds/handlers/fault/v1/handlers/handlers.go

```diff
@@ -692,3 +736,131 @@ func validateTaskNetworkConfig(taskNetworkConfig *state.TaskNetworkConfig) error
 
 	return nil
 }
+
+// startNetworkPacketLossFault invokes the linux TC utility tool to start the network-packet-loss fault.
+func (h *FaultHandler) startNetworkPacketLossFault(taskMetadata *state.TaskResponse, request types.NetworkPacketLossRequest) error {
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
```

Comment:
Can we have a const value for the timeout/5s?

Reply:
Perhaps we can use the timeout constant introduced from my recent PR -> https://github.com/aws/amazon-ecs-agent/pull/4330

Reply:
Addressing in the next revision